### PR TITLE
[Doc] Minor doc fixes for interpolate_bads

### PR DESCRIPTION
**Pull request by mscheltienne**
A couple of minor style fixes while I was looking at the interpolation code.

Before:

Screenshot 2023-01-09 at 18 19 30

After:

Screenshot 2023-01-09 at 18 19 07

If you don't think this is an improvement, feel free to close the PR 😉

### CODE REVIEW

**Comment by larsoner** [comment]
I slightly prefer the old definition-list way of rendering

File: mne/channels/channels.py [hunk 1]
```diff
@@ -1007,16 +1007,17 @@ def interpolate_bads(self, reset_bads=True, mode='accurate',
...
-            - ``"spline"`` (default)
-                Use spherical spline interpolation.
-            - ``"MNE"``
-                Use minimum-norm projection to a sphere and back.
-                This is the method used for MEG channels.
+            * ``"spline"`` (default)
+              Use spherical spline interpolation.
+            * ``"MNE"``
+              Use minimum-norm projection to a sphere and back.
+              This is the method used for MEG channels.
```
Comment (agramfort) [inline mne/channels/channels.py:1016]: I would revert this one

**Comment by mscheltienne** [comment]
Reverted the bullet list style. Render:

Screenshot 2023-01-10 at 17 05 36

File: mne/channels/channels.py [hunk 2]
```diff
@@ -1007,16 +1007,17 @@ def interpolate_bads(self, reset_bads=True, mode='accurate',
             .. versionadded:: 0.17
         method : dict
```
Comment (drammock) [inline mne/channels/channels.py:1008]:
```suggestion
        method : dict | None
```

**Review by drammock — APPROVED** [review]
LGTM pending my one suggestion

**Comment by drammock** [comment]
CircleCI is happy; in it goes! thanks @mscheltienne

**Pull request merged**